ASCOM.Driver/TelescopeDriver/SetupDialogForm.cs

```diff
@@ -929,9 +929,9 @@ private void btnUpdateLoc_Click(object sender, EventArgs e)
 
 			try
 			{
-				_profile.Latitude = System.Convert.ToDouble(txtLat.Text, _oatCulture);
-				_profile.Longitude = System.Convert.ToDouble(txtLong.Text, _oatCulture);
-				_profile.Elevation = Math.Round(System.Convert.ToDouble(txtElevation.Text, _oatCulture));
+				_profile.Latitude = System.Convert.ToDouble(txtLat.Text);
+				_profile.Longitude = System.Convert.ToDouble(txtLong.Text);
+				_profile.Elevation = Math.Round(System.Convert.ToDouble(txtElevation.Text));
```

Comment:
Why remove the` _oatCulture`? If intentional wouldn't be better to keep it safe?

Reply:
Because this is converting user input (from a text input box, which is not in OAT culture, it's in the machine's culture. Only text coming directly from an OAT command reply should use `_oatCulture`, everything else should not specify a culture so that it uses a users system culture.